projects/examples/e2e/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "../../../out-tsc/e2e",
     "module": "commonjs",
-    "target": "es5",
```

Comment:
Why do we change the target?

Reply:
Mistake from early in the upgrade, good catch. Reverted